Pipfile

```diff
@@ -31,6 +31,7 @@ pytest-flask = "~=0.10"
 pytest = "~=3.0"
 pytest-cov = "~=2.7.1"
 coveralls = "~=2.0"
+lovely-pytest-docker = "*"
```

Comment:
I think we need to specify a version range here to avoid nasty surprises?

Reply:
i think this is linked to the pr on top of which this is built, in case better fix there

Reply:
yep, agreed!